sirepo/pkcli/nersc_test.py

```diff
@@ -0,0 +1,51 @@
+# -*- coding: utf-8 -*-
+"""Nerc test suite
+
+:copyright: Copyright (c) 2023 RadiaSoft LLC.  All Rights Reserved.
+:license: http://www.apache.org/licenses/LICENSE-2.0.html
+"""
+from pykern.pkcollections import PKDict
+from pykern.pkdebug import pkdc, pkdexc, pkdp, pkdlog
+from pykern import pkio
```

Comment:
Do fully qualified imports (pykern.pkio, etc.). We are inconsistent, but the default should be qualified.